client/src/js/services/JournalErrorViewerService.js

```diff
@@ -0,0 +1,34 @@
+
+angular.module('bhima.services')
+  .service('JournalErrorViewerService', JournalErrorViewerService);
+
+/** Dependencies injection */
+JournalErrorViewerService.$inject = ['$translate'];
+
+function JournalErrorViewerService($translate) {
+  var service = this;
+
+  service.parseGridRecord = parseGridRecord;
+
+  function parseGridRecord (records){
+    var list = [];
+
+    records.forEach(function (record) {
+      var line = [];
+      var codeTranslated = null;
+      if(record){
```

Comment:
oh yes!!! it is possible. Notice that the record is not the posting journal record but the posting error view.